src/components/shared/Container.tsx

```diff
@@ -0,0 +1,12 @@
+import { FC, ReactNode } from 'react';
+
+import 'styles/common/_base.scss';
+
+interface Props {
+  children: ReactNode | ReactNode[];
+}
+const Container: FC<Props> = ({ children }) => {
```

Comment:
```suggestion
const Container: FC = ({ children }) => {
```
FC include chlidren
remove 
`interface Props {
  children: ReactNode | ReactNode[];
}`

Reply:
This form of recording is incorrect
![image](https://github.com/Egorkostuck/test-task-frontandiya-/assets/69008147/e4dbad34-58d5-47cf-a6f7-cc01534f8e17)
![image](https://github.com/Egorkostuck/test-task-frontandiya-/assets/69008147/cc6cfe84-a7dc-4397-8335-5591ab06af12) 
But there is such a solution :
![image](https://github.com/Egorkostuck/test-task-frontandiya-/assets/69008147/5ab38362-4738-4b15-91c5-90c470f7a963)